my_locustfiles/fund_store.py

```diff
@@ -7,6 +7,8 @@
 class FundStore(HttpUser):
     host = FUND_STORE
     fund_name = "harry-s-breakfast-fund"
+    fund_id = "funding-service-design"
```

Comment:
Both these are actually fund_id parameters - would perhaps be best to just use the line 10 one. (fund_name is a title string like "Harry's Breakfast Fund" not a kebab-case id)

Reply:
@harryyo Have removed fund_name, thus changing some of the tests to use fund_id instead.